control/frdata.py

```diff
@@ -436,12 +440,13 @@ def feedback(self, other=1, sign=-1):
         # TODO: vectorize this
         # TODO: handle omega re-mapping
         for k, w in enumerate(other.omega):
-            fresp[:, :, k] = self.fresp[:, :, k].view(type=matrix)* \
+            fresp[:, :, k] = np.dot(
+                self.fresp[:, :, k],
                 linalg.solve(
-                eye(self.inputs) +
-                other.fresp[:, :, k].view(type=matrix) *
-                self.fresp[:, :, k].view(type=matrix),
-                eye(self.inputs))
+                    eye(self.inputs)
+                    + np.dot(other.fresp[:, :, k], self.fresp[:, :, k]),
+                    eye(self.inputs))
```

Comment:
I know this is not the point of this change, but is  `linalg(X, eye(shape))` any better than `np.linalg.inv`.  I'd have thought a "right divide" (`linalg.solve` with appropriate transposes) would have been a more standard approach.

Reply:
I agree that this could use `sp.linalg.solve` instead, but since I wasn't the original author I just did the conversion to remove the `np.matrix` dependence.  I've added a TODO for someone to fix this, so at least we don't forget about it (in principle).